docs/tutorials/implement-service.md

```diff
@@ -430,7 +437,7 @@ app = FastAPI(
 1. Import the OpenCV library and the get_extension function from the tasks service. This function is used to guess the extension of the image based on the input type.
 2. Change the description of the service.
 3. Change the name and the slug of the service. This is used to identify the service in the Core Engine.
-4. Change the input/output fields of the service. The name of the field is the key of the dictionary that will be used in the process function. The type of the field is the type of the data that will be sent to the service. They are defined in the FieldDescriptionType enum.
+4. Change the input/output fields of the service. The name of the field is the key of the dictionary that will be used in the process function. The type of the field is the type of the data that will be sent to the service. They are defined in the FieldDescriptionType enum. The tags are used to identify the service in the Core Engine. The has_ai variable is used to identify if the service is an AI service.
```

Comment:
I would put the `has_ai` in inline code for a better formatting and understanding:

```markdown
The `has_ai` variable is used to identify if the service is an AI service.
```